### DISCUSSION TITLE
How to use fish as default shell ?

### DISCUSSION THREAD

**Discussion by wiiznokes** [zed-industries/zed#17111] [Feature Requests]
title

**Comment by msehnout** [comment]
settings.json:
```json
{
  "terminal": {
    "shell": {
      "program": "/usr/bin/fish"
    }
  }
}
```

**Reply by msehnout** [reply]
unless your fish binary is located elsewhere, in that case run `where fish`

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
